website/docs/guides/orchestration/webhooks/serverless-datadog.md

```diff
@@ -0,0 +1,111 @@
+---
+title: "Create Datadog events from dbt Cloud results"
+id: webhooks-guide-serverless-datadog
+slug: serverless-datadog
+description: Configure a serverless app to add Datadog logs
+---
+
+This guide will teach you how to build and host a basic Python app which will add dbt Cloud job events to Datadog. To do this, when a dbt Cloud job completes it will create a log entry for each node that was run, containing all information about the node provided by the [Metadata API](docs/dbt-cloud-apis/metadata-schema-models).
```

Comment:
@dpguthrie I'm hassling you non-stop I'm sorry! 

Would love your eyes over these two docs to make sure that they're accurate, and if you have a screenshot of what the logs look like once they're added to Datadog or an example alarm in PagerDuty that would be very helpful for me as well! I haven't actually tested these integrations end to end, I've just got the fly integration up and going and then assumed that the rest of your code is all good.

Reply:
Here's a screenshot for PagerDuty:
![image](https://user-images.githubusercontent.com/10198967/221612581-614603f0-f2f2-4057-8824-86871aa8d449.png)

Let me know if you need something different or more focused.  I just grabbed the entire page for a single alert.

I may be out of luck with the datadog one as my trial has expired.  I'll see what I can dig up though.

The content looks good though - nice work!